### ISSUE TITLE
{Linked:Bug2509192}[Accessibility][Use of color -Workflow Notifications card]: 'View in portal' link is appearing as normal text when grey scale mode is enabled.[Rendering] Bug Title Here

### ISSUE THREAD

**Issue by palitsourish** [microsoft/AdaptiveCards#7716]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS Build: Windows 11
Version: Dev( Build 22526.1000)
Browser: Edge Dev
Version ( 98.0.1108.15) (Official build) (64-bit)
URL: [Designer | Adaptive Cards](https://adaptivecards.io/designer/)
Screen reader: Narrator

Pre-requisites:
Settings > Accessibility > Color Filters > Turn on color filters toggle button > select 'Greyscale' radio button

Repro Steps:
1. Open the above URL and login with valid credentials.
2. Workflow Notifications page will get displayed.
3. Now, enable grey scale mode in settings and navigate to 'View in portal' link in 'company name' section
4. Check whether the link is appearing as normal text or not.

Actual:
"View in portal" link in 'Company name section' is appearing as normal text when grey scale mode is enabled.

Expected:
"View in portal" link in 'Company name section' should not appear as normal text when grey scale mode is enabled.
Links should be easily distinguishable from regular text. Styling options can include underline, a chevron, or a 3:1 contrast difference between the link text and static text (in addition to all text meeting 4:5:1 contrast requirement with the background).

User Impact:
When links appear the same (or similar) to static text most users will have difficulty distinguishing between them which can lead to a frustrating user experience for everyone.

MAS Reference:
[MAS 1.4.1 – Use of Color.](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b63F3625C-8722-40A6-8A2A-64181087C047%7d&file=MAS%201.4.1%20%E2%80%93%20Use%20of%20Color.docx&wdLOR=c82B54E93-B1DC-488E-B43A-A8D659AD7EE4&action=default&mobileredirect=true)

“Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-lalapati%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656401002888328%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=9rKyqggThK5iHRPhG9iMiNp6zSq%2BYr%2fr4gmiXwwFsnM%3D&reserved=0) “

### Screenshots

<img width="1828" alt="MAS1 4 1 'View in portal' link is appearing as normal text when grey scale mode is enabled" src="https://user-images.githubusercontent.com/95914144/181213171-1fcc9404-3f97-4653-9cc9-4fb4ccb53bfb.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue as it's dependent bug is fixed and verified.

Snippet:
![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/3e224f88-aca3-4ae5-8a38-aa47437d1bdb)
